### PR TITLE
[minor change] dhcp labels

### PR DESCRIPTION
**Pull request by akinross**

### CODE REVIEW

File: models/model.go [hunk 1]
```diff
@@ -3,3 +3,19 @@ package models
 type Model interface {
 	ToMap() (map[string]interface{}, error)
 }
+
+type patchPayload struct {
```
Comment (lhercot) [inline models/model.go:7]: PatchPayload

File: models/schema_template_bd.go [hunk 2]
```diff
@@ -51,25 +46,24 @@ func NewTemplateBD(ops, path, name, displayName, layer2Unicast, unkMcastAct, mul
...
-	if bdAttributes.Value != nil {
-		A(bdAttributesMap, "value", bdAttributes.Value)
+func RemoveTemplateBD(path string) *patchPayload {
+	return &patchPayload{
+		Ops:  "remove",
+		Path: path,
 	}
-
-	return bdAttributesMap, nil
 }
```
Comment (lhercot) [inline models/schema_template_bd.go:69]: Make generic and move to models file.

**Review by lhercot — CHANGES_REQUESTED** [review]

File: models/model.go [hunk 3]
```diff
@@ -3,3 +3,26 @@ package models
...
+	patchPayloadAttributesMap := make(map[string]interface{})
+	A(patchPayloadAttributesMap, "op", patchPayloadAttributes.Ops)
+	A(patchPayloadAttributesMap, "path", patchPayloadAttributes.Path)
+	if patchPayloadAttributes.Value != nil {
+		A(patchPayloadAttributesMap, "value", patchPayloadAttributes.Value)
+	}
+	return patchPayloadAttributesMap, nil
+}
+
+func GetRemovePatchPayload(path string) *PatchPayload {
```
Comment (sajagana) [inline models/model.go:23]: I couldn’t understand the usage of GetRemovePatchPayload. Can we use a meaningful name?

**Review by sajagana — APPROVED** [review]
LGTM!

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**